presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -273,6 +272,7 @@ primaryExpression
     | identifier '->' expression                                                     #lambda
     | '(' identifier (',' identifier)* ')' '->' expression                           #lambda
     | '(' query ')'                                                                  #subqueryExpression
+    | EXISTS '(' query ')'                                                           #exists
```

Comment:
What's an example of an expression that doesn't work? Can you add tests to TestSqlParser?